### PR TITLE
[NEW] Set Guest Username Function

### PR DESCRIPTION
**Pull request by AlexanderKanakis**

### CODE REVIEW

File: src/lib/hooks.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ const createOrUpdateGuest = async (guest) => {
 	const { token } = guest;
 	token && await store.setState({ token });
 	token && await loadConfig();
+	console.log('Data');
+	console.log(guest);
```
Comment (ear-dev) [inline src/lib/hooks.js]: Do we want to keep these log points or are they debug?

Comment (AlexanderKanakis) [reply src/lib/hooks.js]: Removing these.

Comment (AlexanderKanakis) [reply src/lib/hooks.js]: @ear-dev Are you sure you are looking at the latest commit? These are removed in the [45f8b80](https://github.com/WideChat/Rocket.Chat.Livechat/pull/127/commits/45f8b80ab3f7a94cfcd41a94b79b62589fe71de3)

Comment (ear-dev) [reply src/lib/hooks.js]: gottit......  sorry about that.

**Review by ear-dev — APPROVED** [review]

**Pull request merged**